source/blog/2018-03-26-ember-3-1-released.md

```diff
@@ -0,0 +1,149 @@
+
+#### Deprecations in Ember 3.1
+
+Deprecations are added to Ember.js when an API will be removed at a later date.
+
+Each deprecation has an entry in the deprecation guide describing the migration
+path to more stable API. Deprecated public APIs are not removed until a major
+release of the framework.
+
+Consider using the
+[ember-cli-deprecation-workflow](https://github.com/mixonic/ember-cli-deprecation-workflow)
+addon if you would like to upgrade your application without immediately addressing
+deprecations.
+
+Two new deprecations are introduces in Ember.js 3.1:
+
+* TODO
+* TODO
+
+For more details on changes in Ember.js 3.1, please review the
+[Ember.js 3.1.0 release page](https://github.com/emberjs/ember.js/releases/tag/v3.1.0).
+
+### Upcoming Changes in Ember.js 3.2
+
+Ember.js 3.2 will introduce two new features:
+
+* TODO
+* TODO
+
+#### Deprecations in Ember.js 3.2
+
+Two new deprecations are introduces in Ember.js 3.2:
+
+* TODO
+* TODO
+
+For more details on the upcoming changes in Ember.js 3.2, please review the
+[Ember.js 3.2.0-beta.1 release page](https://github.com/emberjs/ember.js/releases/tag/v3.2.0-beta.1).
+
+---
+
+## Ember Data
+
+Ember Data is the official data persistence library for Ember.js applications.
+
+### Changes in Ember Data 3.1
+
+#### Deprecations in Ember Data 3.1
+
+Two new deprecations are introduces in Ember Data 3.1:
+
+* TODO
+* TODO
+
+For more details on changes in Ember Data 3.1, please review the
+[Ember Data 3.1.0 release page](https://github.com/emberjs/data/releases/tag/v3.1.0).
+
+
+### Upcoming changes in Ember Data 3.2
+
+
+#### Deprecations in Ember Data 3.2
+
+For more details on the upcoming changes in Ember Data 3.2, please review the
+[Ember Data 3.2.0-beta.1 release page](https://github.com/emberjs/data/releases/tag/v3.2.0-beta.1).
+
+---
+
+## Ember CLI
+
+Ember CLI is the command line interface for managing and packaging Ember.js
+applications.
+
+### Upgrading Ember CLI
+
+You may upgrade Ember CLI separately from Ember.js and Ember Data! To upgrade
+your projects using `yarn` run:
+
+```
+yarn upgrade ember-cli
+```
+
+To upgrade your projects using `npm` run:
+
+```
+npm install --save-dev ember-cli
+```
+
+After running the
+upgrade command run `ember init` inside of the project directory to apply the
+blueprint changes. You can preview those changes for [applications](https://github.com/ember-cli/ember-new-output/compare/v3.0.0...v3.1.0)
+and [addons](https://github.com/ember-cli/ember-addon-output/compare/v3.0.0...v3.1.0).
+
+### Changes in Ember CLI 3.1
+
+#### Deprecations in Ember CLI 3.1
+
+Two new deprecations are introduces in Ember CLI 3.1:
+
+* TODO
+* TODO
+
+For more details on the changes in Ember CLI 3.1 and detailed upgrade
+instructions, please review the [Ember CLI  3.1.0 release page](https://github.com/ember-cli/ember-cli/releases/tag/v3.1.0).
+
+### Upcoming Changes in Ember CLI 3.2
+
+#### Deprecations in Ember CLI 3.2
```

Comment:
- [#7676](https://github.com/ember-cli/ember-cli/pull/7676) Deprecate ember-cli-babel 5.x [@raytiley](https://github.com/raytiley)

Babel 6 support has been out for a long time now and works quite well. Babel 5 support is deprecated and will be dropped soon.